src/main/kotlin/de/bund/digitalservice/useid/tracking/TrackingProperties.kt

```diff
@@ -3,11 +3,9 @@ package de.bund.digitalservice.useid.tracking
 import jakarta.validation.Valid
 import jakarta.validation.constraints.NotBlank
 import org.springframework.boot.context.properties.ConfigurationProperties
-import org.springframework.context.annotation.Profile
 import org.springframework.stereotype.Component
 import org.springframework.validation.annotation.Validated
 
-@Profile("!local")
```

Comment:
Do we need to keep this so that the tests run?

Reply:
No, I removed that so we can send tracking requests locally to our useid-playground matomo site.